apps/transport/test/build_test.exs

```diff
@@ -33,6 +33,19 @@ defmodule TransportWeb.BuildTest do
     {:ok, %{out: "hello\n"}} = Rambo.run("echo", ["hello"])
   end
 
+  test "rambo remains on hex version (not the ARM-compatible fork)" do
+    # NOTE: on ARM architecture (e.g. Mac M1 etc), a local commit can alter `mix.lock`,
+    # and we want to detect cases where this happens. See:
+    # https://github.com/etalab/transport-site/issues/3820
+    # The test has to take the git-stored version of `mix.lock` and assert on its content,
+    # because the CI doesn't run ARM and `mix deps.get` will overwrite `mix.lock` at the moment
```

Comment:
Vraiment pas fan de ce comportement de `mix deps.get` que je réalise à cette occasion du coup…

https://hexdocs.pm/mix/Mix.Tasks.Deps.Get.html

> Gets all out of date dependencies, i.e. dependencies that are not available or have an invalid lock.

[[Proposal] - Add an option to deps.get to force failure on lock mismatch](https://groups.google.com/g/elixir-lang-core/c/Jr2CS6o2elo) serait vraiment bien

Reply:
Oui idem, c'est pas fou, je ne suis pas fan de ça du tout.

J'irai peut-être ajouter notre petit caillou sur le sujet.

Merci d'avoir déterré la conversation sur ce proposal !